src/client.js

```diff
@@ -68,12 +68,15 @@ export class Saturn {
    * @returns {Promise<object>}
    */
   async fetchCIDWithRace (cidPath, opts = {}) {
-    const [cid] = (cidPath ?? '').split('/')
-    CID.parse(cid)
-
-    const jwt = await getJWT(this.opts, this.storage)
+    const options = Object.assign({}, this.opts, opts)
+    if (!opts.originFallback) {
+      const [cid] = (cidPath ?? '').split('/')
+      CID.parse(cid)
+      const jwt = await getJWT(this.opts, this.storage)
+      options.jwt = jwt
+    }
 
-    const options = Object.assign({}, this.opts, { format: 'car', jwt }, opts)
+    options.format = opts.format ? opts.format : 'car'
```

Comment:
This will still return `'car'` when we pass `{ format: null }`. Now i'm wondering if it's better to create a client side only `'flat'` format to avoid falsy checks defaulting to `'car'`, when in fact we want flat files.

Reply:
The test passes for streaming flat files so this does set `{ format: null }` correctly I believe because that is in `opts` and that overwrites the previous object.